src/apps/concepts/components/ConceptsTableRow.tsx

```diff
@@ -90,7 +92,7 @@ const conceptNameCell = (
     >
       <Link
         onClick={(e) => e.stopPropagation()}
-        to={`${row.version_url}?linkedDictionary=${linkedDictionary}`}
+        to={`${row.version_url}?linkedDictionary=${linkedDictionary}&dictionaryToAddTo=${dictionaryToAddTo}&canAddConcept=${buttons.addToDictionary}`}
```

Comment:
It's bad to populate a security concept (`canAddConcept` via a URL parameter)

Reply:
@ibacher  please any suggestion for alternate to that 
how can I pass the `canAddConcept` value

Reply:
You'll basically need to populate this the same way we do [when determining whether a user can modify a dictionary](https://github.com/openmrs/openmrs-ocl-client/blob/a0df00455fbae8484244366e4420389599a80f80/src/apps/concepts/pages/ViewConceptsPage.tsx#L217), i.e., by calling `canModifyContainer()`. `ownerType` and `owner` probably need to be derived from the `dictionaryToAddTo` parameter

Reply:
hi, @ibacher I tried to do the above , But It is not working.
If you say can we encrypt the `canAddConcept` and pass in the same way (as a query parameter).

Reply:
hi @ibacher, @HerbertYiga,  please tell.